<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LancelotAppletConfigBase</class>
 <widget class="QWidget" name="LancelotAppletConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>346</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupAppletIcons">
     <property name="title">
      <string>Applet Icons</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radioShowCategories">
        <property name="text">
         <string>Show cate&amp;gories inside the applet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioShowMenuIconOnly">
        <property name="text">
         <string>Show only &amp;menu launcher icon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedAppletButtons">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pageAppletIconsChoose">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupAppletIconsChoose">
            <property name="title">
             <string>Choose Icon</string>
            </property>
            <layout class="QGridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="iconLancelot">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20000</width>
                 <height>64</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="iconKDE">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20000</width>
                 <height>64</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="iconStartHere">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20000</width>
                 <height>64</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="KIconButton" name="iconCustom">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20000</width>
                 <height>64</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="icon">
                <string>run</string>
               </property>
               <property name="iconSize">
                <number>48</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="radioIconLancelot">
               <property name="text">
                <string>&amp;Lancelot</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="radioIconKDE">
               <property name="text">
                <string>&amp;KDE Logo</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QRadioButton" name="radioIconStartHere">
               <property name="text">
                <string>&amp;Start Here</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QRadioButton" name="radioIconCustom">
               <property name="text">
                <string>Custo&amp;m</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageCategoriesChoose">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupCategoriesChoose">
            <property name="title">
             <string>Choose categories to show</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QListWidget" name="listSections">
               <property name="styleSheet">
                <string>.QListWidget {
    background: rgba(0,0,0,0)
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="viewMode">
                <enum>QListView::IconMode</enum>
               </property>
               <property name="uniformItemSizes">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupActivation">
     <property name="title">
      <string>Menu Activation</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radioActivationHover">
        <property name="text">
         <string>Show menu on icon &amp;hover (after a predefined interval of time)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioActivationClick">
        <property name="text">
         <string>Show menu on &amp;click</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>iconCustom</sender>
   <signal>iconChanged(QString)</signal>
   <receiver>radioIconCustom</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateCard()</slot>
 </slots>
</ui>
